modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -206,3 +206,13 @@ extension Set: RequestDecodable where Element: Content {
 extension Set: Content where Element: Content { }
 
 extension AnyCodable: Content {}{{/useVapor}}
+
+extension JSONEncodable where Self: Encodable {
+    func encodeToJSON() -> Any {
+        let encoder = JSONEncoder()
+        guard let data = try? encoder.encode(self) else {
+            fatalError("Could not encode to json: \(self)")
+        }
+        return data.base64EncodedString(options: Data.Base64EncodingOptions())
```

Comment:
@0x0c I'm a little worried about this.
When do we want to encode json as base64?
Only in multipart/form-data requests?
Can't we just return the json in `Data`?

Reply:
@4brunu According to generated code, `Data` can encode to json as follows.
```
extension Data: JSONEncodable {
    func encodeToJSON() -> Any {
        return self.base64EncodedString(options: Data.Base64EncodingOptions())
    }
}
```
This code is implemented [here](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/swift5/Extensions.mustache#L60). Therefore, I followed this code and encode json as base 64.

Reply:
I see.
Could you please reuse that extension please?
`return data.encodeToJSON()`

Reply:
I updated line 216 so please check https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/7836e277f06208d5da3ce5bf1adb86f15bed6e71